_includes/auth/gen-rsa-key-pairs.md

```diff
@@ -63,17 +63,13 @@ sudo zypper install openssl
 
 #### Create a private/public key pair on Linux
 
-To create a private/public key pair:
-
-1. On a command line, run:
-
-    ```sh
-    ssh-keygen -m PEM -t rsa -b 2048 -f <MY_KEY_PAIR_NAME>
-    ```
+{% include auth/recommend-passphrase.md %}
```

Comment:
Hi @spocke, I've added a notice to the relevant sections, _partly_ for legal reasons. We need to be clear that this is not ideal for security, and do not want to be encouraging poor practices.

Reply:
Is it though. We generate a unique private/public key pair by cryptographically random generation we tell them to keep the private key safe. Adding a passphrase to my mind adds no extra security since you would have to pass that passphrase in anyway and have it stored possibly together with the key. You might be able to have a setup where the key is stored in one place then the passphrase in another but they would still need to come together to produce a signed JWT key and that would be the attack surface where you would have both things anyway. To me passphrases is more if you use the key for login purposes where you want a user to enter that passphrase in some secure context. @tjdett might know more about this.

Reply:
I've had a think on this, and i may have a better solution that better addresses both our concerns. I'll revert my commit and make fresh edits tomorrow.

Reply:
Ok,so I've revised the approach. Rather than tell them to add or not add a passphrase, we will indicate that the passphrase is optional and follow it with a note indicating why.